Run.hx

```diff
@@ -1,13 +1,244 @@
+						'php.files.${c}';
+					// These require phpnatives (https://lib.haxe.org/p/phpnatives/)
+					case 'ImapCloseFlags' | 'ImapHeaders' | 'ImapOverview' | 'ImapSearchFlags' |
+					'ImapSortFlags' | 'ImapStatusFlags' | 'ImapStream' | 'MailAddress' |
+					'MailboxInfo' | 'MailStructure':
+						'php.imap.${c}';
+					// These require phpnatives (https://lib.haxe.org/p/phpnatives/)
+					case 'Countable' | 'Iterator' | 'OuterIterator' | 'RecursiveIterator' |
+					'SeekableIterator' | 'Serializable' | 'SplObserver' | 'SplSubject':
+						'php.interfaces.${c}';
+					// These require phpnatives (https://lib.haxe.org/p/phpnatives/)
+					case 'AppendIterator' | 'ArrayIterator' | 'CachingIterator' |
+					'CallbackFilterIterator' | 'DirectoryIterator' | 'EmptyIterator' |
+					'FilesystemIterator' | 'FilterIterator' | 'GlobIterator' | 'InfiniteIterator' |
+					'IteratorIterator' | 'LimitIterator' | 'MultipleIterator' | 
+					'NoRewindIterator' | 'ParentIterator' | 'RecursiveArrayIterator' |
+					'RecursiveCachingIterator' | 'RecursiveCallbackFilterIterator' |
+					'RecursiveDirectoryIterator' | 'RecursiveFilterIterator' |
+					'RecursiveIteratorIterator' | 'RecursiveRegexIterator' |
+					'RecursiveTreeIterator' | 'RegexIterator':
+						'php.iterators.${c}';
+					// These require phpnatives (https://lib.haxe.org/p/phpnatives/)
+					case 'ArrayObject':
+						'php.misc.${c}';
+					// These require phpnatives (https://lib.haxe.org/p/phpnatives/)
+					case 'DOMAttr' | 'DOMCdataSection' | 'DOMCharacterData' | 'DOMComment' |
+					'DOMDocument' | 'DOMDocumentFragment' | 'DOMDocumentType' | 'DOMElement' |
+					'DOMEntity' | 'DOMEntityReference' | 'DOMException' | 'DOMImplementation' |
+					'DOMNamedNodeMap' | 'DOMNode' | 'DOMNodeList' | 'DOMNotation' |
+					'DOMProcessingInstruction' | 'DOMText' | 'DOMXPath' | 'LibXMLError' |
+					'SimpleXMLElement' | 'SimpleXMLIterator' | 'XMLReader' | 'XSLTProcessor':
+						'php.xml.${c}';
+					// All other/unknown classes
+					case _:
+						'${basePackage}.${c}';
+				}
+			case _:
+				t.getName();
+		};
+	}
+
+	static function getConst(c:peg.php.PConst):String {
+		return '${c.visibility} const ${c.name}:${getType(c.type)}';
+	}
+
+	static function getVarName(name:String):String {
+		return ~/^[$]/.replace(name, '');
+	}
+
+	static function getVar(v:peg.php.PVar):String {
+		return '${v.visibility} var ${getVarName(v.name)}:${getType(v.type)}';
+	}
+
+	static function getFunction(fn:peg.php.PFunction, isNamespaceGlobal:Bool = false, ?namespace:String):String {
+		var args = fn.args.map(arg -> '${getVarName(arg.name)}:${getType(arg.type)}').join(', ');
+		var inlineCallParams = fn.args.map(arg -> getVarName(arg.name)).join(', ');
+		var callSite = '';
+		if (isNamespaceGlobal) {
+			var ns = namespace != '' ? '\\\\${namespace}\\\\': '';
+			callSite = ' return untyped __call__(\'${ns}${fn.name}\'${inlineCallParams != '' ? ', ' + inlineCallParams : ''})';
```

Comment:
use `php.Syntax.call` instead of `untyped __call__`

Reply:
done in 09f1f4c